WooCommerce/Classes/ViewModels/Order Details/Receipts/ReceiptEligibilityUseCase.swift

```diff
@@ -44,25 +48,53 @@ final class ReceiptEligibilityUseCase: ReceiptEligibilityUseCaseProtocol {
         stores.dispatch(action)
     }
 
+    /// Returns true if Point of Sale allows sending successful payment email receipts via the API.
+    /// WooCommerce 9.5 allows to attach a customer email after payment is made and send email receipt via the API.
+    ///
     func isEligibleForPointOfSaleReceipts(onCompletion: @escaping (Bool) -> Void) {
         guard featureFlagService.isFeatureFlagEnabled(.sendReceiptsForPointOfSale) else {
             onCompletion(false)
             return
         }
 
         Task { @MainActor in
-            async let isWooCommerceSupported = isPluginSupported(Constants.wcPluginName,
+            let isWooCommerceSupported = await isPluginSupported(Constants.wcPluginName,
```

Comment:
Interesting one, I sort of thought that these were the same, but they aren't 🤔 
- `async let` creates the task to execute the function, and while will run async it does not await the result immediately, so it runs concurrently to other stuff we may have within the scope, and its non-blocking.
- await runs sequentially and it's blocking, as awaits immediately and must complete before moving to the next line.